tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+from click.testing import CliRunner
+
+
+@pytest.fixture()
+def build_resources(tmpdir):
```

Comment:
think you could add some one-liner docstrings for these for others to quickly know what they're for?